.github/workflows/test-python.yml

```diff
@@ -205,12 +205,11 @@ jobs:
             python-version: '3.8'
         - name: Start MongoDB
           uses: supercharge/mongodb-github-action@1.10.0
-        - name: Run Test
+        - name: Run connect test from sdist
           shell: bash
           run: |
             cd sdist/test
             ls
             which python
             pip install -e ".[test]"
-            pytest -v
-            pytest -v test/asynchronous/
+            PYMONGO_MUST_CONNECT=1 pytest -v test/test_client_context.py
```

Comment:
Has this test been just running the wrong suite this whole time?

Reply:
No but it just seems unnessecary to run the whole suite from the sdist, we already run the whole suite elsewhere on the same commit.